src/Util/UrlBuilder.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace QuizApp\Util;
+
+/**
+ * Class UrlBuilder
+ * @package QuizApp\Util
+ */
+class UrlBuilder
+{
+
+    /**
+     * UrlBuilder constructor.
+     */
```

Comment:
Do we really need this method declaration? Remove if not pls.

Reply:
It was auto-generated by PhpStorm